ui/src/App.js

```diff
@@ -121,10 +122,32 @@ class App extends Component {
     datasetApi.datasetGet(datasetCallback);
   }
 
-  handleChange(selectedOption) {
+  handleChange(selectedFacets) {
```

Comment:
Since we use "extraFacets" elsewhere, let's call this extraFacetsDicts.
```
# extraFacetsDicts is a list of dicts, where each dict is:
{
  'label': field name - field description,
  'value': Elasticsearch field name
}
```

Reply:
Done